src/main/scala/org/allenai/plugins/VersionInjectorPlugin.scala

```diff
@@ -40,6 +44,18 @@ object VersionInjectorPlugin extends AutoPlugin {
       "gitDescribe",
       "The description of the current git commit"
     )
+    val gitMRC = TaskKey[String](
```

Comment:
This name should match the other hash-generating task (`gitSha1`). Maybe `gitLocalSha1`?